features/poller/monitors/core/blueprint-poller.xml

```diff
@@ -0,0 +1,33 @@
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
+	xmlns:ext="http://aries.apache.org/blueprint/xmlns/blueprint-ext/v1.1.0"
+	xsi:schemaLocation="
+		http://www.osgi.org/xmlns/blueprint/v1.0.0 
+		http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd
+		http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0
+		http://aries.apache.org/schemas/blueprint-cm/blueprint-cm-1.1.0.xsd
+		http://aries.apache.org/blueprint/xmlns/blueprint-ext/v1.1.0
+		http://aries.apache.org/schemas/blueprint-ext/blueprint-ext-1.1.xsd
+">
+
+	<bean id="poller" class="org.opennms.netmgt.poller.remote.Poller"/>
+	
+	<bean id="availabilityMonitor" class="org.opennms.netmgt.poller.monitors.AvailabilityMonitor" />
+
+	<!-- ActiveMQ component provided by minion-core -->
+	<reference id="queuingservice" interface="org.apache.camel.Component" filter="(alias=opennms.broker)"/>
+
+	<camelContext id="pollerCamelContext" xmlns="http://camel.apache.org/schema/blueprint">
+<!-- 		<propertyPlaceholder id="properties" location="blueprint:discovererProperties" /> -->
+
+		<!-- Need to do it for the all the monitors -->
+		<!-- Availability Monitor -->
+		<route id="pollAvailabilityMonitor">
+			<from uri="queuingservice:Location-localhost" />
+			<setExchangePattern pattern="InOut"/>
+			<to uri="bean:availabilityMonitor?method=poll" />
```

Comment:
I think you might need to write a processor class here because the poll() method takes two arguments, the MonitoredService and the Map of parameters.